bin/start_fe.sh

```diff
@@ -115,6 +115,10 @@ if [[ "$JAVA_VERSION" -gt 8 ]]; then
     final_java_opt=$JAVA_OPTS_FOR_JDK_9
 fi
 
+if [[ "$JAVA_VERSION" -lt 11 ]]; then
+    echo "tips: current jdk version is $JAVA_VERSION, using jdk 11 or higher will provide better performance"
```

Comment:
Better to give more reason why higher JDK can provide better performance.

Reply:
added
better GC performance.